django/db/backends/mysql/client.py

```diff
@@ -8,11 +8,11 @@ class DatabaseClient(BaseDatabaseClient):
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
         args = [cls.executable_name]
         env = None
-        db = settings_dict['OPTIONS'].get('db', settings_dict['NAME'])
+        database = settings_dict['OPTIONS'].get('database', settings_dict['NAME'])
```

Comment:
We should support both `db` and `database`, e.g.
```python
database = settings_dict['OPTIONS'].get(
    'database',
    settings_dict['OPTIONS'].get('db', settings_dict['NAME']),
)
```

Reply:
🤦🏻‍♂️ Yes.